addon/utils/mung-options-for-fetch.ts

```diff
@@ -13,7 +12,7 @@ import {
 export default function mungOptionsForFetch(
   options: AjaxOptions
 ): FetchOptions {
-  const hash = assign(
+  const hash = Object.assign({},
```

Comment:
I think we may need to drop support for ember < 4 to do this as 3.28 still supports IE 11

Reply:
would you like to me to drop support for ember < 4 in this PR ?

Reply:
Is that true 🤔 this just moves out of the concern of the addon and could be handled by a [babel plugin](https://babeljs.io/docs/babel-plugin-transform-object-assign) or something

Reply:
essentially it would be bad if we dropped support for 3.28 for no reason

Reply:
ember-fetch does not work on Ember 5 without this change

Reply:
sorry I wasn't 100% clear 🙈 we should make this change, but we shouldn't stop testing in Ember 3.28

Reply:
I've updated the code.
I had to remove support for `Ember < 3.28` because of `ember-qunit` but that's about it.
So this PR now supports Ember >= 3.28 (including Ember 5).
I've refactored the code so that neither the `ember assign polyfill` nor `Object.assign()` are used in production code. So IE is also still supported.